src/main/target/ALIENWHOOP/config.c

```diff
@@ -97,58 +107,85 @@ void targetConfiguration(void)
 
     compassConfigMutable()->mag_hardware =  MAG_NONE;
 
+    systemConfigMutable()->cpu_overclock = 2; //216MHZ
+
     /* Default to 32kHz enabled at 16/16 */
     gyroConfigMutable()->gyro_use_32khz = 1; // enable 32kHz sampling
     gyroConfigMutable()->gyroMovementCalibrationThreshold = 200; // aka moron_threshold
     gyroConfigMutable()->gyro_sync_denom = 2;  // 16kHz gyro
     pidConfigMutable()->pid_process_denom = 1; // 16kHz PID
+    gyroConfigMutable()->gyro_lowpass_type = FILTER_PT1;
```

Comment:
There are a lot of values set in here that are simply copies of the existing default. This is not sustainable for code maintenance. Please remove.

Reply:
@mikeller , I understand. The idea behind explicitly defining filters and loop times is to ensure future target performance remains the same, for example when the defaults are changed. For example if the default gyro lowpass filter is changed to biquad that could have a very dramatic effect. It seems like there is still debate over the defaults. These loop times and filters seem to perform well and are very much interdependent. Perhaps my logic is flawed. :grin: I will try to go through and find which are defaults and remove them tonight.

Reply:
If anything is changed away from what are good defaults for this craft you can always add a line to `targetConfiguration()` to revert to the old default - we only release once a month, so there will time for the change to go into the next release.
Getting a CI build failure, and then having to go and update half a dozen target configurations that re-define the default gets old very quickly...